Nodejs/Product/TestAdapter/TestFrameworks/Jest/jestReporter.js

```diff
@@ -0,0 +1,32 @@
+﻿class jestReporter {
```

Comment:
does this work with node 6? and Node 8?

Reply:
Yes, check https://node.green/#ES2015-functions-class-class-statement if you'll like some details.